ygot/types.go

```diff
@@ -58,6 +56,17 @@ type ValidatedGoStruct interface {
 	ΛBelongingModule() string
 }
 
+// validatedGoStruct is an interface used for validating GoStructs.
+type validatedGoStruct interface {
```

Comment:
Essentially, this interface is internal-only such that it is *always* implemented by a generated struct, right?

Reply:
Yes, `ΛValidate` and `IsYANGGoStruct()` are always generated.

Reply:
Great, thanks -- can we note this in the description?

Reply:
```
 59 // validatedGoStruct is an interface used for validating GoStructs.                                                                                                                                                                                                                                                                                    
 60 // This interface is implemented by all Go structs (YANG container or lists),                                                                                                                                                                                                                                                                          
 61 // regardless of generation flag.                                                                                                                                                                                                                                                                                                                      
```
Done.